sdk/formrecognizer/Azure.AI.FormRecognizer/src/ExtractedTableCell.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+
+namespace Azure.AI.FormRecognizer.Models
+{
+    public class ExtractedTableCell
+    {
+        internal ExtractedTableCell(DataTableCell_internal dataTableCell, ReadResult_internal readResult, ICollection<string> references)
+        {
+            BoundingBox = new BoundingBox(dataTableCell.BoundingBox);
+            ColumnIndex = dataTableCell.ColumnIndex;
+            ColumnSpan = dataTableCell.ColumnSpan.HasValue ? dataTableCell.ColumnSpan.Value : 1;
```

Comment:
You can write it as `ColumnSpan = dataTableCell.ColumnSpan ?? 1` same for booleans